packages/vite/src/node/cli.ts

```diff
@@ -51,6 +51,22 @@ function cleanOptions<Options extends GlobalCLIOptions>(
   return ret
 }
 
+function filterDuplicateOptions<Options extends GlobalCLIOptions>(
+  options: Options
+) {
+  if (options && typeof options === 'object') {
+    Object.entries(options).forEach((item) => {
+      if (Array.isArray(item[1])) {
+        const filter_value = [...new Set(item[1])]
+        //@ts-ignore
+        options[item[0]] =
+          filter_value.length === 1 ? filter_value[0] : filter_value
```

Comment:
If it's an array, I think we can simply take the last one and assign it. So it'd be the same logic as #10474

Reply:
Yes, so I'll close this pr.